src/webapi/event.rs

```diff
@@ -453,3 +1085,55 @@ reference_boilerplate! {
     instanceof Event
     convertible to Event
 }
+
+impl ErrorEvent {
+    /// Returns a human-readable error message describing the problem.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/message)
+    #[inline]
+    pub fn message( &self ) -> String {
+        return js!(
+            @{self.as_ref()}.message;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the name of the script where the error occurred.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/filename)
+    #[inline]
+    pub fn filename( &self ) -> String {
+        return js!(
+            @{self.as_ref()}.filename;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the line number of the script file where the error occurred.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/lineNo)
+    #[inline]
+    pub fn line_no( &self ) -> i32 {
```

Comment:
As far as I can see [here](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent) the property is named `lineno`, so to keep with the convention there shouldn't be an underscore in this function's name.

Ditto for `colno`.